drivers/storage/libstorage/libstorage_client_api.go

```diff
@@ -255,6 +261,16 @@ func (c *client) VolumeAttach(
 	request *types.VolumeAttachRequest) (*types.Volume, string, error) {
 
 	ctx = c.withInstanceID(c.requireCtx(ctx), service)
+	ctxA, err := c.withAllLocalDevices(ctx)
```

Comment:
Hi @clintonskitson,

It looks like one of these should be WithAllInstanceIDs. I'm handling it on the equivalent PR I'm submitting, but just noting it here.